### PR TITLE
Create Ansible role to add learner-portal MFE into sandbox builds

### PR DESCRIPTION
**Pull request by Tj-Tracy**
Configuration Pull Request
---

Make sure that the following steps are done before merging:

  - [ ] A DevOps team member has approved the PR if it is code shared across multiple services and you don't own all of the services.
  - [ ] Are you adding any new default values that need to be overridden when this change goes live? If so:
    - [ ] Update the appropriate internal repo (be sure to update for all our environments)
    - [ ] If you are updating a secure value rather than an internal one, file a DEVOPS ticket with details.
    - [ ] Add an entry to the CHANGELOG.
  - [x] If you are making a complicated change, have you performed the proper testing specified on the [Ops Ansible Testing Checklist](https://openedx.atlassian.net/wiki/display/EdxOps/Ops+Ansible+Testing+Checklist)?  Adding a new variable does not require the full list (although testing on a sandbox is a great idea to ensure it links with your downstream code changes).
  - [ ] Think about how this change will affect Open edX operators.  Have you updated the wiki page for the next Open edX release?

As a note - this work is to support the master's team getting a workable sandbox. This PR should be very similar to https://github.com/edx/configuration/pull/5260

### CODE REVIEW

File: playbooks/learner_portal.yml [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+  gather_facts: True
+  vars:
+    ENABLE_NEWRELIC: False
+    CLUSTER_NAME: 'learner-portal'
+    LEARNER_PORTAL_ENABLED: True
+    LEARNER_PORTAL_SANDBOX_BUILD: False
+  roles:
+    - aws
+    - role: nginx
+      nginx_app_dir: "/etc/nginx"
```
Comment (schenedx) [inline playbooks/learner_portal.yml]: I don't think we need this specification. It should just be the default.

File: playbooks/edx_continuous_integration.yml [hunk 2]
```diff
@@ -37,6 +37,8 @@
       when: REGISTRAR_ENABLED
     - role: journals
       when: JOURNALS_ENABLED
+    - role: learner_portal
```
Comment (schenedx) [inline playbooks/edx_continuous_integration.yml:42]: I mentioned this before, but I will mention it here. I believe you need to add `learner_portal` role into the nginx service list after line 14 for learner-portal to be served out of the box on sandbox builds.

File: playbooks/roles/learner_portal/tasks/main.yml [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+  git:
+    dest: "{{ learner_portal_code_dir }}"
+    repo: "{{ learner_portal_repo }}"
+    version: "{{ learner_portal_version }}"
+    accept_hostkey: yes
+  become_user: "{{ learner_portal_user }}"
+  register: learner_portal_checkout
+
+- name: Install nodeenv
+  command: apt install nodeenv
```
Comment (schenedx) [inline playbooks/roles/learner_portal/tasks/main.yml]: You can just use the `apt` module to do this. It's super easy

**Review by schenedx — CHANGES_REQUESTED** [review]
Some minor feedback. I think this is close.

File: playbooks/edx_continuous_integration.yml (hunk 2 above)
Comment (Tj-Tracy) [reply playbooks/edx_continuous_integration.yml:42]: Whoops, I thought I had made this change. Thanks for pointing it out. Though the conversation wasn't updated, I've made the change.

**Review by schenedx — APPROVED** [review]
👍

**Pull request merged**